core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -352,6 +354,13 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       }
     }
 
+  def getTags(query: Option[String]): Future[PerRequestMessage] =
+    dataSource.inTransaction { dataAccess =>
+      dataAccess.workspaceQuery.getTags(query).map { result =>
+        RequestComplete(StatusCodes.OK, result.sortWith(_.count > _.count))
```

Comment:
I'd prefer to move the sorting down into the SQL (see previous comment), but if that's a problem I'm ok with it here.